Assets/Scripts/AILookBehaviour.cs

```diff
@@ -0,0 +1,90 @@
+using System.Collections;
+using UnityEngine;
+
+public enum LookActionType
+{
+    Turn, Wait
+}
+
+[System.Serializable]
+public class LookAction
+{
+    public LookActionType actionType;
+
+    public Vector3 targetPosition;
+    public float timeDuration;
+}
+
+public class AILookBehaviour : MonoBehaviour
```

Comment:
I don't think we will use this anywhere else other than for Camera movement, so we can have the naming be based on CameraLookBehaviour or something similar.